.github/workflows/darker.yml

```
@@ -17,7 +17,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.12'
+          python-version: "3.12"
       - uses: akaihola/darker@1.7.2
         with:
-          version: "1.7.2"
+          version: "2.1.1" # This will use the 2.1.1 darker Python package while using the 1.7.2 darker github action
```

I'm surprised that this works, I would have assumed the version has to be < the tag in the "uses" entry. In any case why do we not change both spots?

```suggestion
      - uses: akaihola/darker@2.1.1
        with:
          version: "2.1.1"
```

---

I was also expecting this `      - uses: akaihola/darker@2.1.1` to work as I tried it earlier. But darker did not publish the github actions for akaihola/darker@2.1.1 and the last one is akaihola/darker@1.7.2. Although the last python package they published was 2.1.1.

---

What is the latest gh action version? It's unclear to me then that the usage of `version: "2.1.1"` is having any effect. 

We can update to whatever the latest gh action is instead of 2.1.1 for both the pre-commit and the gh action. It's best for those to be using the same version.

---

Digging more into this. The github actions is reading from github branches. And the naming convention was updated to include `v` at the start of the version number.
I update now.